### PR TITLE
[Y2K-334] redo recommendations once contacts are loaded

### PR DESCRIPTION
**Pull request by jzila**
cc @keybase/y2ksquad

### CODE REVIEW

File: shared/team-building/index.tsx [hunk 1]
```diff
@@ -86,10 +86,11 @@ const ContactsBanner = ({onRedoSearch}: {onRedoSearch: () => void}) => {
   const onLater = React.useCallback(() => dispatch(SettingsGen.createImportContactsLater()), [dispatch])
   // Redo search if # of imported contacts changes
   React.useEffect(() => {
-    if (prevNumContactsImported !== undefined && prevNumContactsImported !== numContactsImported) {
+    if (prevNumContactsImported !== numContactsImported) {
```
Comment (buoyad) [inline shared/team-building/index.tsx]: We should keep `prevNumContactsImported !== undefined &&`, it avoids a false positive on mount.

Comment (jzila) [reply shared/team-building/index.tsx]: Doesn't it always start as undefined until we import the contacts? I'll put it back in and test, I guess.

Comment (buoyad) [reply shared/team-building/index.tsx]: On mount `numContactsImported === null` and `prevNumContactsImported === undefined`
https://github.com/keybase/client/blob/1227eeb259d67a5bd35ceef507be69370ca60d6b/shared/constants/settings.tsx#L90-L95

Comment (jakob223) [reply shared/team-building/index.tsx]: 🤯

Comment (jzila) [reply shared/team-building/index.tsx]: @buoyad got it, thanks.

**Pull request merged**